app/policies/request_policy.rb

```diff
@@ -40,15 +36,17 @@ def complete?
   def destroy?
     return false unless grower?
 
-    return true if record.pending?
-
-    record.errors.add(:status, 'can\'t delete a non-pending request')
-    false
+    if record.pending?
+      true
+    else
+      record.errors.add(:status, 'can\'t delete a non-pending request')
+      false
+    end
```

Comment:
simple remarque de style, mais on écrirait plutôt ça comme en ruby : 
```suggestion
    return true if record.pending?

    record.errors.add(:status, 'can\'t delete a non-pending request')
    false
```
(en gros, on évite les if/else si un simple if qui met fin à la méthode suffit ! Ça allège la lecture et la "charge mentale" de la logique :) )